sdk/src/test/java/com/uid2/sdk/extensions/StringExTest.kt

```diff
@@ -0,0 +1,34 @@
+package com.uid2.sdk.extensions
+
+import org.json.JSONObject
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class StringExTest {
+
+    @Test
+    fun `test parsing json to map`() {
+        val map = mapOf(
+            "Key1" to "Value1",
```

Comment:
Only testing string:string maps here.... but the signature is:
```
decodeJsonToMap(): Map<String, Any>?
```
So we might want more test cases?

Reply:
As mentioned earlier, I believe this method will be removed "soon", so will leave as is (if okay)